src/python/pants/core/register.py

```diff
@@ -71,6 +72,7 @@ def rules():
         *export_codegen_goal.rules(),
         *fmt.rules(),
         *fix.rules(),
+        *python_lockfile.rules(),
```

Comment:
I don't think the `core` rules should be registering the `python` rules. What's this for?